uber/configspec.ini

```diff
@@ -49,6 +49,9 @@ at_the_con = boolean(default=False)
 # and behaviors such as automated emails which are ONLY turned on afterwards.
 post_con = boolean(default=False)
 
```

Comment:
Two things:

1) We should be more explicit in this comment about what is being disabled.  I would expect "disables any internet-dependent features" to mean that we're not trying to send emails and that it's impossible to make Stripe payments, but that doesn't seem to actually be the case.

2) Relatedly, we probably should disable those features if this is set.  Although since we already have a separate setting for enabling/disabling email, perhaps we should instead make that setting have its default value be the value of this setting.